frog/imports/ui/repository/Activity.jsx

```diff
@@ -20,44 +20,46 @@ export default class Activity extends Component {
   }
 
   renderObjectProperties(object, key) {
-        class Property extends Component {
-          render() {
-            return(
-              <div>
-                <br/>
-                <a>{this.props.data.prop + ": " + this.props.data.value}</a>
-              </div>
-            );
-          }
-        }
-
-        return (Object.keys(object).map((prop) =>(
-          <Property key={prop} data = {{prop:prop, value:object[prop]}} />
-        )));
+
+    class Property extends Component {
+      render() {
+        return(
+          <div>
+          <br/>
+          <a>{this.props.data.prop + ": " + this.props.data.value}</a>
+          </div>
+          );
+      }
+    }
+
+    return (Object.keys(object).map((prop) =>(
+      <Property key={prop} data = {{prop:prop, value:object[prop]}} />
+      )));
+
   }
 
 
   render() {
     return (
       <div className="activity-summary">
       <li onClick={this.activityHandler.bind(this)}>
-        {this.props.name}, type:{this.props.type}, plane:{this.props.plane}
+      {this.props.name}, type:{this.props.type}, plane:{this.props.plane}
       </li>
```

Comment:
I disagree with this indentation change. Everything line between `<li>` and `</li`> should have +2 spaces indent so that we see right away where the list ends. What do you think ?